x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/ransomware.tsx

```diff
@@ -0,0 +1,364 @@
+              {
+                defaultMessage: 'Notify User',
+              }
+            )}
+          />
+        </>
+        {userNotificationSelected && (
+          <>
+            <EuiSpacer size="s" />
+            <EuiFlexGroup gutterSize="s">
+              <EuiFlexItem grow={false}>
+                <EuiText size="xs">
+                  <h4>
+                    <FormattedMessage
+                      id="xpack.securitySolution.endpoint.policyDetailsConfig.customizeUserNotification"
+                      defaultMessage="Customize notification message"
+                    />
+                  </h4>
+                </EuiText>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiIconTip
+                  position="right"
+                  data-test-subj="ransomwareTooltip"
+                  content={
+                    <>
+                      <FormattedMessage
+                        id="xpack.securitySolution.endpoint.policyDetailsConfig.ransomware.notifyUserTooltip.a"
+                        defaultMessage="Selecting the user notification option will display a notification to the host user when ransomware is prevented or detected."
+                      />
+                      <EuiSpacer size="m" />
+                      <FormattedMessage
+                        id="xpack.securitySolution.endpoint.policyDetailsConfig.ransomware.notifyUserTooltip.b"
+                        defaultMessage="
+                    The user notification can be customized in the text box below. Bracketed tags can be used to dynamically populate the applicable action (such as prevented or detected) and the filename."
+                      />
+                    </>
+                  }
+                />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+            <EuiSpacer size="xs" />
+            <EuiTextArea
+              placeholder={i18n.translate(
+                'xpack.securitySolution.endpoint.policyDetails.ransomware.userNotification.placeholder',
+                {
+                  defaultMessage: 'Input your custom notification message',
+                }
+              )}
+              value={userNotificationMessage}
+              onChange={handleCustomUserNotification}
+              fullWidth={true}
+              data-test-subj="ransomwareUserNotificationCustomMessage"
+            />
+          </>
+        )}
+      </>
+    );
+  }, [
+    radios,
+    selected,
+    handleUserNotificationCheckbox,
+    userNotificationSelected,
+    userNotificationMessage,
+    handleCustomUserNotification,
+  ]);
+
+  const protectionSwitch = useMemo(() => {
+    return (
+      <EuiSwitch
+        label={i18n.translate(
+          'xpack.securitySolution.endpoint.policy.details.ransomwareProtectionsEnabled',
+          {
+            defaultMessage:
+              'Ransomware Protections {mode, select, true {Enabled} false {Disabled}}',
```

Comment:
check with Bonnie if all the first letter of words in this sentence should be capitalized. or should it really be:

```suggestion
              'Ransomware protections {mode, select, true {enabled} false {disabled}}',
```